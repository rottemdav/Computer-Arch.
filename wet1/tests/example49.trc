2 7 2 0 global_history global_tables using_share_lsb
0x36227c T 0x5efe8d0
0x1ab8f7c T 0x6128f488
0x8ee60 N 0x778b378
0x8ee60 N 0x778b378
0xf7317c N 0x13c
0x36227c T 0x5efe8d0
0x99ec N 0x57112c
0x4c50938 N 0xb3347898
0x1ab8f7c T 0x6128f488
0x4c50938 N 0xb3347898
0xf7317c T 0x13c
0x8ee60 T 0x778b378
0x36227c T 0x5efe8d0
0xe1c N 0x4724
0x8ee60 N 0x778b378
0x99ec T 0x57112c
0x1ab8f7c N 0x6128f488
0xf7317c N 0x13c
0x4c50938 T 0xb3347898
0xf7317c T 0x13c
0x1ab8f7c T 0x6128f488
0x1ab8f7c N 0x6128f488
0x36227c T 0x5efe8d0
0xe1c T 0x4724
0x99ec T 0x57112c
0x36227c N 0x5efe8d0
0x1ab8f7c T 0x6128f488
0x8ee60 N 0x778b378
0x99ec N 0x57112c
0x4c50938 N 0xb3347898
0xf7317c N 0x13c
0x4c50938 N 0xb3347898
0x99ec T 0x57112c
0xe1c N 0x4724
0x99ec N 0x57112c
0x1ab8f7c N 0x6128f488
0x99ec N 0x57112c
0x1ab8f7c T 0x6128f488
0x99ec T 0x57112c
0x1ab8f7c T 0x6128f488
0xe1c N 0x4724
0x99ec N 0x57112c
0x99ec N 0x57112c
0xe1c N 0x4724
0xf7317c T 0x13c
0xe1c T 0x4724
0x8ee60 N 0x778b378
0x1ab8f7c T 0x6128f488
0x8ee60 N 0x778b378
0x1ab8f7c N 0x6128f488
0x8ee60 N 0x778b378
0x99ec N 0x57112c
0x1ab8f7c T 0x6128f488
0xf7317c N 0x13c
0xe1c N 0x4724
0xe1c T 0x4724
0x99ec N 0x57112c
0xe1c N 0x4724
0x4c50938 T 0xb3347898
0x8ee60 T 0x778b378
0xe1c N 0x4724
0xf7317c T 0x13c
0x4c50938 T 0xb3347898
0x1ab8f7c T 0x6128f488
0x8ee60 T 0x778b378
0x8ee60 T 0x778b378
0x99ec N 0x57112c
0xf7317c N 0x13c
0xe1c N 0x4724
0xe1c N 0x4724
0xe1c T 0x4724
0x1ab8f7c N 0x6128f488
0x36227c N 0x5efe8d0
0xf7317c T 0x13c
0x8ee60 T 0x778b378
0x99ec N 0x57112c
0x99ec N 0x57112c
0xe1c T 0x4724
0xf7317c T 0x13c
0x36227c N 0x5efe8d0
0x4c50938 N 0xb3347898
0x36227c T 0x5efe8d0
0xe1c N 0x4724
0x4c50938 T 0xb3347898
0xe1c N 0x4724
0x36227c T 0x5efe8d0
0x1ab8f7c N 0x6128f488
0x1ab8f7c N 0x6128f488
0xf7317c T 0x13c
0x36227c T 0x5efe8d0
0x36227c T 0x5efe8d0
0x36227c N 0x5efe8d0
0x99ec N 0x57112c
0xe1c N 0x4724
0x1ab8f7c T 0x6128f488
0x8ee60 T 0x778b378
0xe1c T 0x4724
0x99ec T 0x57112c
0x4c50938 N 0xb3347898
0x99ec T 0x57112c
0xf7317c T 0x13c
0x8ee60 T 0x778b378
0xe1c N 0x4724
0x36227c N 0x5efe8d0
0x36227c T 0x5efe8d0
0x1ab8f7c T 0x6128f488
0xf7317c N 0x13c
0xe1c T 0x4724
0x8ee60 T 0x778b378
0x1ab8f7c T 0x6128f488
0x8ee60 T 0x778b378
0x4c50938 N 0xb3347898
0xf7317c T 0x13c
0x36227c N 0x5efe8d0
0x99ec T 0x57112c
0x1ab8f7c T 0x6128f488
0x1ab8f7c N 0x6128f488
0x8ee60 N 0x778b378
0xe1c T 0x4724
0x4c50938 T 0xb3347898
0xe1c N 0x4724
0x36227c T 0x5efe8d0
0x36227c T 0x5efe8d0
0x99ec N 0x57112c
0x4c50938 N 0xb3347898
0x99ec N 0x57112c
0x8ee60 N 0x778b378
0x8ee60 T 0x778b378
0x1ab8f7c N 0x6128f488
0x36227c N 0x5efe8d0
0x1ab8f7c N 0x6128f488
0x36227c N 0x5efe8d0
0x8ee60 T 0x778b378
0x1ab8f7c T 0x6128f488
0x36227c N 0x5efe8d0
0x99ec T 0x57112c
0x36227c T 0x5efe8d0
0x1ab8f7c T 0x6128f488
0xf7317c T 0x13c
0xe1c N 0x4724
0x4c50938 N 0xb3347898
0x4c50938 N 0xb3347898
0x1ab8f7c T 0x6128f488
0x99ec N 0x57112c
0xe1c N 0x4724
0xf7317c T 0x13c
0x1ab8f7c T 0x6128f488
0x4c50938 T 0xb3347898
0x99ec T 0x57112c
0xe1c N 0x4724
0xe1c T 0x4724
0x4c50938 T 0xb3347898
0x8ee60 T 0x778b378
0x4c50938 T 0xb3347898
0x99ec T 0x57112c
0x99ec T 0x57112c
0x36227c T 0x5efe8d0
0x8ee60 N 0x778b378
0xf7317c T 0x13c
0x36227c N 0x5efe8d0
0x1ab8f7c T 0x6128f488
0x99ec N 0x57112c
0x36227c T 0x5efe8d0
0xf7317c N 0x13c
0xf7317c T 0x13c
0x99ec T 0x57112c
0x1ab8f7c N 0x6128f488
0xf7317c N 0x13c
0x8ee60 T 0x778b378
0x8ee60 N 0x778b378
0x99ec N 0x57112c