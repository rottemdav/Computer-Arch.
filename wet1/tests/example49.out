0x36227c N 0x362280
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0x8ee60 N 0x8ee64
0xf7317c N 0xf73180
0x36227c N 0x362280
0x99ec N 0x99f0
0x4c50938 N 0x4c5093c
0x1ab8f7c N 0x1ab8f80
0x4c50938 N 0x4c5093c
0xf7317c N 0xf73180
0x8ee60 N 0x8ee64
0x36227c N 0x362280
0xe1c N 0xe20
0x8ee60 N 0x8ee64
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0xf7317c N 0xf73180
0x4c50938 N 0x4c5093c
0xf7317c N 0xf73180
0x1ab8f7c N 0x1ab8f80
0x1ab8f7c N 0x1ab8f80
0x36227c N 0x362280
0xe1c N 0xe20
0x99ec N 0x99f0
0x36227c N 0x362280
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0x99ec N 0x99f0
0x4c50938 N 0x4c5093c
0xf7317c N 0xf73180
0x4c50938 N 0x4c5093c
0x99ec N 0x99f0
0xe1c N 0xe20
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0xe1c N 0xe20
0x99ec N 0x99f0
0x99ec N 0x99f0
0xe1c N 0xe20
0xf7317c N 0xf73180
0xe1c N 0xe20
0x8ee60 N 0x8ee64
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0xf7317c N 0xf73180
0xe1c N 0xe20
0xe1c N 0xe20
0x99ec N 0x99f0
0xe1c N 0xe20
0x4c50938 N 0x4c5093c
0x8ee60 N 0x8ee64
0xe1c N 0xe20
0xf7317c N 0xf73180
0x4c50938 N 0x4c5093c
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0x8ee60 N 0x8ee64
0x99ec N 0x99f0
0xf7317c N 0xf73180
0xe1c N 0xe20
0xe1c N 0xe20
0xe1c N 0xe20
0x1ab8f7c N 0x1ab8f80
0x36227c N 0x362280
0xf7317c N 0xf73180
0x8ee60 N 0x8ee64
0x99ec N 0x99f0
0x99ec N 0x99f0
0xe1c N 0xe20
0xf7317c N 0xf73180
0x36227c N 0x362280
0x4c50938 N 0x4c5093c
0x36227c N 0x362280
0xe1c N 0xe20
0x4c50938 N 0x4c5093c
0xe1c N 0xe20
0x36227c N 0x362280
0x1ab8f7c N 0x1ab8f80
0x1ab8f7c N 0x1ab8f80
0xf7317c N 0xf73180
0x36227c N 0x362280
0x36227c N 0x362280
0x36227c N 0x362280
0x99ec N 0x99f0
0xe1c N 0xe20
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0xe1c N 0xe20
0x99ec N 0x99f0
0x4c50938 N 0x4c5093c
0x99ec N 0x99f0
0xf7317c N 0xf73180
0x8ee60 N 0x8ee64
0xe1c N 0xe20
0x36227c N 0x362280
0x36227c N 0x362280
0x1ab8f7c N 0x1ab8f80
0xf7317c N 0xf73180
0xe1c N 0xe20
0x8ee60 N 0x8ee64
0x1ab8f7c N 0x1ab8f80
0x8ee60 T 0x778b378
0x4c50938 N 0x4c5093c
0xf7317c N 0xf73180
0x36227c N 0x362280
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0x1ab8f7c N 0x1ab8f80
0x8ee60 N 0x8ee64
0xe1c N 0xe20
0x4c50938 N 0x4c5093c
0xe1c N 0xe20
0x36227c N 0x362280
0x36227c N 0x362280
0x99ec N 0x99f0
0x4c50938 N 0x4c5093c
0x99ec N 0x99f0
0x8ee60 N 0x8ee64
0x8ee60 N 0x8ee64
0x1ab8f7c N 0x1ab8f80
0x36227c N 0x362280
0x1ab8f7c N 0x1ab8f80
0x36227c N 0x362280
0x8ee60 N 0x8ee64
0x1ab8f7c N 0x1ab8f80
0x36227c N 0x362280
0x99ec N 0x99f0
0x36227c N 0x362280
0x1ab8f7c T 0x5efe8d0
0xf7317c N 0xf73180
0xe1c N 0xe20
0x4c50938 N 0x4c5093c
0x4c50938 N 0x4c5093c
0x1ab8f7c N 0x1ab8f80
0x99ec N 0x99f0
0xe1c N 0xe20
0xf7317c N 0xf73180
0x1ab8f7c N 0x1ab8f80
0x4c50938 N 0x4c5093c
0x99ec N 0x99f0
0xe1c N 0xe20
0xe1c N 0xe20
0x4c50938 N 0x4c5093c
0x8ee60 N 0x8ee64
0x4c50938 N 0x4c5093c
0x99ec N 0x99f0
0x99ec N 0x99f0
0x36227c N 0x362280
0x8ee60 N 0x8ee64
0xf7317c N 0xf73180
0x36227c N 0x362280
0x1ab8f7c N 0x1ab8f80
0x99ec N 0x99f0
0x36227c N 0x362280
0xf7317c N 0xf73180
0xf7317c N 0xf73180
0x99ec N 0x99f0
0x1ab8f7c N 0x1ab8f80
0xf7317c N 0xf73180
0x8ee60 N 0x8ee64
0x8ee60 N 0x8ee64
0x99ec N 0x99f0
flush_num: 86, br_num: 171, size: 329b
